core/domain/user_services_test.py

```diff
@@ -998,6 +1003,7 @@ def utcnow(cls):
 
 
 class LastExplorationEditedIntegrationTests(test_utils.GenericTestBase):
+    """Integration tests for the exploration that is edited at the last."""
```

Comment:
I am not sure this is right. Could you confirm? I have to look through the code. Will check and let you know in the next review pass.

Reply:
```suggestion
    """Integration tests for testing the time the user last updated an exploration updates correctly."""
```

Reply:
Done